src/hooks/useGatewayArioInfo.ts

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from '@tanstack/react-query';
+
+export interface ArioInfoResponse {
+  wallet: string;
+  processId: string;
+  // ans104UnbundleFilter:...
+  // ans104IndexFilter:...
```

Comment:
Are these comments intended?

Reply:
Yes, I'm not sure of the data schema for these values and they're not being used, so I opted to leave them as comments for now.